README.md

```diff
@@ -146,13 +146,13 @@ Feel free to look around!
 We now need to pull the Elixir dependencies:
 
 ```
-mix deps.get
+$ mix deps.get
```

Comment:
This `$` might make things clearer, but it's way more annoying to copy. I don't think it's worth it - where else will people copy command lines?

Reply:
It's a matter of start copying from "mix" rather than double clicking (which is dangerous, because of double click will copy newline and it will make a command execute on paste), and some commands require manually changing values anyway.

IMHO preventing people from blinding copy & pasting should considered a good thing.

(Actually, have you noticed that all internal repo has `$` prefix in command examples?)